src/components/calculator.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Please do not disable linters, they are there to help you. Remember that writing a good code is very beneficial.

Reply:
Hello, thanks for the review. I don't think there is any other thing I can do to this linter error. We were asked to build with classes and the error is telling I have to change it to a stateless function. We will be instructed in the next milestone to change it to a stateless function but not this one.
Do you have any suggestions?

Reply:
Hello @fobadara, thank you for letting me know. Just remove the commented out code and ask for another code review